Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseOpMode.java

```diff
@@ -245,23 +256,26 @@ public void centerJunctionTop(GrabberCameraPipeline pipeline) {
      * @param pipeline the RobotCameraPipeline being used by the robot camera
      */
     public void centerConeStack(RobotCameraPipeline pipeline) {
-        double xOffset, width;
-
+        double xOffset, width, strafePower, turnPower;
         do {
-            xOffset = pipeline.xPosition - Constants.CAMERA_CENTER_X;
+            xOffset = Constants.CAMERA_CENTER_X-pipeline.xPosition;
             width = pipeline.width;
 
             // drive forward while centering on the cone stack if contour exists
             if (width == 0) {
                 break;
             } else {
-                driveWithIMU(coneStackPixelsMotorPower(xOffset), coneStackWidthMotorPower(width), 0.0);
-                telemetry.addData("xMotorPower", coneStackPixelsMotorPower(xOffset));
+                strafePower = coneStackPixelsAndWidthToStrafingPower(xOffset, width);
+                turnPower = 1.09*coneStackPixelsAndWidthToTurningPower(xOffset, width);
```

Comment:
1.09 is a magic number. Please use a constant. Also why do you even need it? Shouldn't the method give you the correct value?

Reply:
Constant created

Reply:
I don't see the constant